packages/terra-dropdown-button/src/Item.jsx

```diff
@@ -52,7 +52,9 @@ const Item = ({
         });
       }}
       tabIndex="0"
-      role="menuitem"
+      role="option"
+      aria-selected={false} // Needed to allow VoiceOver on iOS to announce "selected" state
+      aria-checked={false} // Needed to allow JAWS to announce "selected" state
```

Comment:
```suggestion
      aria-selected={false} // Needed to allow VoiceOver on iOS to announce "selected" state
      aria-checked={false} // Needed to allow JAWS to announce "selected" state
```
Do you still need them as you are falling back to custom messages for conveying selection states

Reply:
To announce list-items as 'not selected' I have added this.

Reply:
Is these behaviour / input provided on Jira by UX Team

Reply:
Checked with UX team, As suggested not required to announce the 'not selected' state have reverted those aria-properties.